### ISSUE TITLE
Mapping events supported from DataAggregator library 

### ISSUE THREAD

**Issue by lapistano** [liip/LiipDataAggregatorBundle#7]
There are already a number of events thrown by the DataAggregator library.
It would be a great idea to make them recognized by SF2 as well.

**Comment by dbu** [comment]
is this using the symfony/event-dispatcher component? then it would probably simply be a question of adjusting the configuration of the bundle to inject the default event dispatcher, right?

**Comment by lapistano** [comment]
@dbu no, it is currently not using the SF2 Event-Dispatcher, and probably won't - I think it is just too much for this lib ... nevertheless I think the DataAggregatorBundle should use it and just reroute Events thrown by the lib to the dispatcher. 
That said, I still have not made my mind up on how to do it.

**Comment by dbu** [comment]
i guess the best would be an adapter that wraps the symfony event dispatcher and implements the DataAggregator event dispatcher so that you can inject that.

that said, the symfony event dispatcher is not particularly heavy to use, why not switch to that one rather than reimplement one?

**Comment by lapistano** [comment]
need to investigate ....
